third_party/ui/bower_components/angular-material/modules/closure/select/select.js

```diff
@@ -757,7 +757,7 @@ function SelectProvider($$interimElementProvider) {
       var focusedNode = centeredNode || optionNodes[0];
       if (focusedNode) {
         opts.focusedNode = focusedNode;
-        focusedNode.focus();
+        // focusedNode.focus();
```

Comment:
Why is this commented out? Will it ever be uncommented? What's the TODO here? (and elsewhere)